language/diem-framework/modules/DiemConsensusConfig.move

```diff
@@ -0,0 +1,78 @@
+address 0x1 {
+
+
+/// Maintains the consensus config for the Diem blockchain. The config is stored in a
+/// DiemConfig, and may be updated by Diem root.
+module DiemConsensusConfig {
+    use 0x1::CoreAddresses;
+    use 0x1::DiemConfig::{Self, DiemConfig};
+    use 0x1::DiemTimestamp;
+    use 0x1::Roles;
+    use 0x1::Vector;
+
+    struct DiemConsensusConfig has copy, drop, store {
+        config: vector<u8>,
+    }
+
+    /// Publishes the DiemConsensusConfig config.
+    public fun initialize(
+        dr_account: &signer,
+    ) {
+        Roles::assert_diem_root(dr_account);
+        DiemConfig::publish_new_config<DiemConsensusConfig>(
```

Comment:
I don't think providing the type argument here does much, nor does it really increase the readability for me, I'd remove it personally